From patchwork Mon Nov 26 15:21:18 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Paolo Bonzini <pbonzini@redhat.com>
X-Patchwork-Id: 10698497
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 15CB113BB
	for <patchwork-kvm@patchwork.kernel.org>;
 Mon, 26 Nov 2018 15:21:25 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 067D329EB3
	for <patchwork-kvm@patchwork.kernel.org>;
 Mon, 26 Nov 2018 15:21:25 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id EECC729EDC; Mon, 26 Nov 2018 15:21:24 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 79CCC29ECD
	for <patchwork-kvm@patchwork.kernel.org>;
 Mon, 26 Nov 2018 15:21:24 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1726266AbeK0CPs (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Mon, 26 Nov 2018 21:15:48 -0500
Received: from mail-wm1-f67.google.com ([209.85.128.67]:40919 "EHLO
        mail-wm1-f67.google.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1726248AbeK0CPs (ORCPT <rfc822;kvm@vger.kernel.org>);
        Mon, 26 Nov 2018 21:15:48 -0500
Received: by mail-wm1-f67.google.com with SMTP id q26so18838349wmf.5
        for <kvm@vger.kernel.org>; Mon, 26 Nov 2018 07:21:22 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=20161025;
        h=sender:from:to:cc:subject:date:message-id;
        bh=HcNiW4mmJMzWSB3TxhQo88KTRBd7EJ3EgODa76//304=;
        b=jNCpYD6KW36kQZeyQaE/vC9n78Mp+S1NtombULxiup32c58mYqESl9sl/zbNrRpGiV
         xm8KbLBgLaXvxOAd9kLaZHuUEJE6l0X0glmK2RPC0lf9+pBgLvDj5LVXhb8XhBVWh/mJ
         BW0FTnvSXkZ0RONHF8lHOsuH7sv81KMICQAf1DFpVCSGWxrD4LdYUOcOZQiXgzHG7MK3
         yN0IMCHIzNtD42e+uKj2D/UGE3PtpxuFrnoDY2+E72Ab13JJeKQ8LyQqGIEpOccezDdx
         TckfVnFJl3cEQqB3DzpxAHSN+nDY4MivTbtLxjPdxxXFPwBJvD1VFVmCbgYbtmA1THZv
         6k5w==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20161025;
        h=x-gm-message-state:sender:from:to:cc:subject:date:message-id;
        bh=HcNiW4mmJMzWSB3TxhQo88KTRBd7EJ3EgODa76//304=;
        b=JOjeqzGP52jW5+6hwE4g0/XMjBv0PEb7lC5BBHutrzJM2L4z3MwvABOsFmcgCNz3MV
         2JisQOHOOL6cQAor4Y1dMXiZlmmKkVHc1FOkHStXq6nbkYdzvL+8GfZAGpXIBGoB44kq
         v0S9UCu5aAoa2x6NROFwsc1SKxBDa0MepNemJT1dETnRnlWrGlJkApf8/mDz1laZ5Sv0
         OLxc4cbwwqOU3ndxt39iQy0+CbL2LpaSaqO/vmiagZMi48eia/+yJdExwVAtduQlYUeG
         j4xSop+/DkeiJCJVUWroY4exvWin6a7kwTCuxzDUm77LwJpQIuWB1POdlp5v68u3osIJ
         rtig==
X-Gm-Message-State: AGRZ1gLE4yOtC0WIeAm3LzBRg/OwclIi4Zz3fM+CkGsSR3eOacKFDvxJ
        e6vUeN/mmnm15qltTbN74muXtoY9
X-Google-Smtp-Source: 
 AFSGD/UPaAlQi1PAjwlgcsUoPT2czIYw3EVKpNaRfi2HfiyNgdUShqpyxs6y8lgPuPq3YFzEjK90Cw==
X-Received: by 2002:a1c:a8cf:: with SMTP id
 r198mr22890155wme.95.1543245680975;
        Mon, 26 Nov 2018 07:21:20 -0800 (PST)
Received: from 640k.lan ([93.56.166.5])
        by smtp.gmail.com with ESMTPSA id s9sm444182wrw.26.2018.11.26.07.21.19
        (version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Mon, 26 Nov 2018 07:21:20 -0800 (PST)
From: Paolo Bonzini <pbonzini@redhat.com>
To: kvm@vger.kernel.org
Cc: lcapitulino@redhat.com, drjones@redhat.com
Subject: [PATCH kvm-unit-tests] README: document ACCEL environment variable
Date: Mon, 26 Nov 2018 16:21:18 +0100
Message-Id: <1543245678-48085-1-git-send-email-pbonzini@redhat.com>
X-Mailer: git-send-email 1.8.3.1
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

Suggested-by: Luiz Capitulino <lcapitulino@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
Reviewed-by: David Hildenbrand <david@redhat.com>
Reviewed-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 README.md | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/README.md b/README.md
index 453eb85..1a9a4ab 100644
--- a/README.md
+++ b/README.md
@@ -47,6 +47,11 @@ environment variable:
 
     QEMU=/tmp/qemu/x86_64-softmmu/qemu-system-x86_64 ./x86-run ./x86/msr.flat
 
+To select an accelerator, for example "kvm" or "tcg", specify the
+ACCEL=<name> environment variable:
+
+    ACCEL=kvm ./x86-run ./x86/msr.flat
+
 # Unit test inputs
 
 Unit tests use QEMU's '-append <args...>' parameter for command line
